frontend/src/dashboard/components/DashCard.jsx

```diff
@@ -44,7 +44,16 @@ export default class DashCard extends Component {
         let error = (dataset && dataset.error) || this.state.error;
 
         if (error) {
-            let message = (error.data && error.data.message) || error;
+            let message;
+            if (error.data) {
+                message = error.data.message
+            } else if (error.status === 503) {
+                message = "I'm sorry, the server timed out while asking your question."
```

Comment:
this seems inconsistent with the handling in the QB.  don't we need to ensure that at least 15s has elapsed before considering this a timout?  otherwise it should be considered a server issue right?